modules/aruco/include/opencv2/aruco/charuco.hpp

```diff
@@ -42,6 +42,8 @@ the use of this software, even if advised of the possibility of such damage.
 #include <opencv2/core.hpp>
```

Comment:
many changes -> can we put the new shorter license header?

Reply:
fixed

added shorter license to all modified files